<launch>

    <node name="UWB_VIO_Landing" pkg="offboard_control" type="landing" respawn="true" output="screen">
        <rosparam file="$(find offboard_control)/config/landing_params.yaml" command="load"/>
    </node>

    <!-- <node name="DWM1001_Active" pkg="dwm1001_uwb_tag_drivers" type="dwm1001_active" respawn="true" output="screen">
        <rosparam file="$(find offboard_control)/config/uwb_params.yaml" command="load"/>
    </node> -->

</launch>